src/Ticket.php

```diff
@@ -6003,6 +6003,7 @@ public static function cronCreateInquest($task)
         }
 
         foreach ($tabentities as $entity => $rate) {
+            $current       = Entity::getUsedConfig('inquest_config', $entity, 'id');
```

Comment:
This will not return the current entity ID. I think `$entity` corresponds to the value you are trying to get here.